charts/netbox/templates/deployment.yaml

```diff
@@ -105,8 +105,26 @@ spec:
         - name: http
           containerPort: 8080
           protocol: TCP
-        {{- if .Values.readinessProbe.enabled }}
-        readinessProbe:
+        - name: nginx-status
+          containerPort: 8081
+          protocol: TCP
+        {{- if .Values.customLivenessProbe }}
+        livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.customLivenessProbe "context" $) | nindent 10 }}
+        {{- else if .Values.livenessProbe.enabled }}
+        livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.livenessProbe "enabled") "context" $) | nindent 10 }}
+          httpGet:
+            path: /{{ .Values.basePath }}api/status/
```

Comment:
Turns out that as of NetBox 4.0.2, because of the CVE, `LOGIN_REQUIRED` now defaults to `True` and `/api/status/` is no longer reachable.

Maybe we should just hit `/api/` and then add some documentation that if you set `LOGIN_REQUIRED = False` then you can point at `/api/status/` instead?

Reply:
It's a bit complicated to test for now, since the `LOGIN_REQUIRED` is not yet `True` as default in current docker images.
It seems not to be released yet: https://github.com/netbox-community/netbox-docker/compare/netbox-community:0c99ff8...netbox-community:15da2f0

Reply:
@RangerRick Is it possible to reconsider? As `LOGIN_REQUIRED` is not yet `True` in current container images, these probes still make sense.
We can review the situation as soon as the new image default are published, with a proper testing environment. 

What do you think?

Reply:
@LeoColomb Hrm, for security reasons we honestly probably should be changing our default to `true` in `values.yaml` too.

Seems like for now the best bet is to leave the `/login/` check alone, but add in your enhancement to check the nginx bit.

Long-term I'd rather we figure out a way to do something more robust that does "real" checking, whatever that means. Right now this just feels like moving from one not-so-great option to a different slightly-less-bad option.

Reply:
```suggestion
            path: /{{ .Values.basePath }}login/
```